milestone3/frontend1/src/App.js

```diff
@@ -3,77 +3,42 @@ import AboutMe from "./aboutMe";
 import { BrowserRouter, Route, Routes } from "react-router-dom";
 import Home from "./home";
 import Main from "./main";
-import React from "react";
 import RecipePage from "./recipePage";
-import recipes from "./recipeData.js";
+import React, { useState, useEffect } from 'react';
 
 function App() {
+  let [allRecipes, setRecipe] = useState();
+  useEffect(() => {
+    const loadRecipe = async () => {
+      let res = await fetch('http://localhost:3001/api/recipe')
+      setRecipe(await res.json())
+    }
+    loadRecipe([]);
+  }, [])
+
   return (
     <BrowserRouter>
       <Routes>
-        <Route path="/allRecipes.html" element={<Home />} />
+        <Route path="/allRecipes.html" element={<Home allRecipes={allRecipes} />} />
         <Route path="/main.html" element={<Main />} />
```

Comment:
I would suggest having the path of your main be just "/".
This can be a catchall for broken routes (So if they go to localhost:3001/somebadURL, they will be redirected to your main page!